<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cal_presets</class>
 <widget class="QDialog" name="cal_presets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration presets</string>
  </property>
  <widget class="QComboBox" name="cal_kit_combo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>85033E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CS5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibration kit:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="full_2port_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Full 2-Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trl_2port_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2-Port TRL</string>
   </property>
  </widget>
  <widget class="QLabel" name="channel_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of channels:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="channel_combo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>52</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
